spring-ai-spring-boot-autoconfigure/pom.xml

```diff
@@ -281,6 +282,12 @@
 			<optional>true</optional>
 		</dependency>
 
+		<dependency>
```

Comment:
What is the reason to add the elasticsearch-java explicitly here? As it is already defined in the vector-store dependenies?